specs/commandline/verify.md

```diff
@@ -113,13 +124,46 @@ An example of output messages for a successful verification:
 Successfully verified signature for localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
 ```
 
+### Verify signatures on an OCI artifact with user metadata
+
+Use the `--user-metadata` flag to verify that provided key-value pairs are present in the payload of the valid signature.
+
+```shell
+# Verify signatures on the supplied OCI artifact identified by the digest and verify that io.wabbit-networks.data=foo is present in the signed payload
+notation verify localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9 --user-metadata io.wabbit-networks.data=foo
```

Comment:
What will happen if `--user-metadata` was not specified in `notation verify` command, however the metadata was added during `notation sign`

Reply:
if `--user-metadata` is not specified in `notation verify`, then metadata won't be verified as part of overall signature verification even if the metadata is present.

Reply:
I think we need to document this behavior in the signature spec.  Any comments @priteshbandi @shizhMSFT

Reply:
No, user metadata is signature envelope agnostic, it will always be string-string key value pair.

Reply:
@priteshbandi  I mean should we document this behavior "if --user-metadata is not specified in notation verify, then metadata won't be verified as part of overall signature verification even if the metadata is present."

Reply:
If we're already saying to use `--user-metadata` to verify key value pairs, is it necessary to also specify the opposite? (if you don't specify `--user-metadata` then we won't verify)